### DISCUSSION TITLE
Does the scale ready only when a weight is first added to it, or intermittently?

### DISCUSSION THREAD

**Discussion by mhendu** [dckiller51/bodymiscale#50] [Q&A]
Trying to figure out if I could place a water tank on the scale and periodically obtain readings of its weight to know if the tank is full or not. The Mi Flora devices give periodic reports on soil / moisture data, wondering if the scale works the same way or if it only reports when someone first steps on it.

**Comment by dckiller51** [comment]
> Trying to figure out if I could place a water tank on the scale and periodically obtain readings of its weight to know if the tank is full or not. The Mi Flora devices give periodic reports on soil / moisture data, wondering if the scale works the same way or if it only reports when someone first steps on it.

Hello, the components does not recover bluetooth data. this is done by solutions like esphome ... But to answer your question the scale goes to sleep after 5 minutes so it would not work to recover the weight continuously.
